skore/src/skore/_sklearn/_plot/metrics/precision_recall_curve.py

```diff
@@ -321,7 +321,7 @@ def _plot_cross_validated_estimator(
                         "average_precision"
                     ]
                     average_precision_mean = np.mean(average_precision)
-                    average_precision_std = np.std(average_precision)
+                    average_precision_std = np.std(average_precision, ddof=1)
```

Comment:
There is a discrepancy between `np.std` and `series.std` where the `ddof` is set to 0 in NumPy and 1 in Pandas.

I made it consistent by detecting the bug. We align with `ddof=1`.